### DISCUSSION TITLE
Cálculo PWR

### DISCUSSION THREAD

**Discussion by HectorLestani** [GArCyAR/ReunionesAnuales#2] [Ideas]
En este espacio proponemos discutir estrategias, ideas y comentarios sobre cómo cubrir la necesidad de herramientas de cálculo propias para núcleos PWR.

**Comment by gtheler** [comment]
El IAEA 3D PWR benchmark resuelto con FeenoX explotando la simetría 1/8vo a partir de un CAD 3D:

https://seamplex.com/feenox/examples/neutron_diffusion.html#iaea-3d-pwr-benchmark
